std_msgs/Header header
uint8 state
MotorVel vel

# MotorVel vel
# # float32 left
# # float32 right